src/main/java/com/unbappd/enterprise/EnterpriseApplicationController.java

```diff
@@ -10,7 +10,7 @@
      * @return
      */
     @RequestMapping("/")
-    public String index() {
+    public int index() {
 
         return "index";
     }
```

Comment:
I changed the variable from string to int, so it will be more consistent.

Reply:
Don't do this.  This breaks the program, and will result in a compile error.
- @RequestMapping should return either a String, which represents the next page in the flow, or a ModelAndView.
- Look at the return type on line 15.  It's a String in quotes.  If the return type is an int, this will break.

Broken code == more technical debt.